xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-api/src/main/java/org/xwiki/livedata/internal/LiveDataScriptServiceConfiguration.java

```diff
@@ -44,16 +44,16 @@
 import org.xwiki.livedata.LiveDataQuery;
 
 /**
- * Provides services to manipulate configurations for the {@link LiveDataRendererConfiguration}.
+ * Provides services to manipulate configurations for the {@link LiveDataScriptServiceConfiguration}.
  *
  * @version $Id$
  * @since 14.9
  * @since 14.4.7
  * @since 13.10.10
  */
-@Component(roles = LiveDataRendererConfiguration.class)
+@Component(roles = LiveDataScriptServiceConfiguration.class)
 @Singleton
-public class LiveDataRendererConfiguration
+public class LiveDataScriptServiceConfiguration
```

Comment:
Since this code is not inside a `script` directory, it's not meant to be used only by script services and thus the name is wrong to me (or the location).